mock/pubSubTopicStub.go

```diff
@@ -0,0 +1,38 @@
+package mock
```

Comment:
Not needed for IsInterfaceNil ?

Reply:
this one too has the main implementation on libp2p, thus no IsInterfaceNil